src/views/Request.svelte

```diff
@@ -15,13 +15,17 @@ $: request = $requests.find(({ id }) => id === params.id) || {}
 $: requester = request.createdBy || {}
 $: isMine = $me.id && (requester.id === $me.id) // Check $me.id first to avoid `undefined === undefined`
 $: destination = request.destination && request.destination.description || ''
+$: origin = request.origin && request.origin.description || ''
```

Comment:
you like the parens around the first logical block in another file so it be good to be consistent.  I personally like this approach better but understand if you want them.